bigframes/core/indexes/base.py

```diff
@@ -30,6 +30,7 @@
 import bigframes.core.expression as ex
 import bigframes.core.ordering as order
 import bigframes.core.utils as utils
+from bigframes.core.validate import enforce_ordered, requires_strict_ordering
```

Comment:
Import modules, not methods.

https://google.github.io/styleguide/pyguide.html#22-imports

There are only a few exceptions: https://google.github.io/styleguide/pyguide.html#2241-exemptions

Reply:
done